conf/pipeline/eager/mpcdf.config

```diff
@@ -1,64 +1,121 @@
 
-  withName: malt {
-    maxRetries = 1
-    memory = 725.GB
-    cpus = 40
-    time = 24.h
-  }
-  
-  withLabel:'sc_tiny'{
-    cpus = { check_max( 1, 'cpus' ) }
-    memory = { check_max( 1.GB * task.attempt, 'memory' ) }
-    time = 24.h
-  }
+      withName: malt {
+        maxRetries = 1
+        memory = 725.GB
+        cpus = 40
+        time = 24.h
+      }
+      
+      withLabel:'sc_tiny'{
+        cpus = { check_max( 1, 'cpus' ) }
+        memory = { check_max( 1.GB * task.attempt, 'memory' ) }
+        time = 24.h
+      }
 
-  withLabel:'sc_small'{
-    cpus = { check_max( 1, 'cpus' ) }
-    memory = { check_max( 4.GB * task.attempt, 'memory' ) }
-    time = 24.h
-  }
+      withLabel:'sc_small'{
+        cpus = { check_max( 1, 'cpus' ) }
+        memory = { check_max( 4.GB * task.attempt, 'memory' ) }
+        time = 24.h
+      }
 
-  withLabel:'sc_medium'{
-    cpus = { check_max( 1, 'cpus' ) }
-    memory = { check_max( 8.GB * task.attempt, 'memory' ) }
-    time = 24.h
-  }
+      withLabel:'sc_medium'{
+        cpus = { check_max( 1, 'cpus' ) }
+        memory = { check_max( 8.GB * task.attempt, 'memory' ) }
+        time = 24.h
+      }
 
-  withLabel:'mc_small'{
-    cpus = { check_max( 2 * task.attempt, 'cpus' ) }
-    memory = { check_max( 4.GB * task.attempt, 'memory' ) }
-    time = 24.h
-  }
+      withLabel:'mc_small'{
+        cpus = { check_max( 2 * task.attempt, 'cpus' ) }
+        memory = { check_max( 4.GB * task.attempt, 'memory' ) }
+        time = 24.h
+      }
 
-  withLabel:'mc_medium' {
-    cpus = { check_max( 4 * task.attempt, 'cpus' ) }
-    memory = { check_max( 8.GB * task.attempt, 'memory' ) }
-    time = 24.h
-  }
+      withLabel:'mc_medium' {
+        cpus = { check_max( 4 * task.attempt, 'cpus' ) }
+        memory = { check_max( 8.GB * task.attempt, 'memory' ) }
+        time = 24.h
+      }
 
-  withLabel:'mc_large'{
-     cpus = { check_max( 8 * task.attempt, 'cpus' ) }
-     memory = { check_max( 16.GB * task.attempt, 'memory' ) }
-     time = 24.h
-  }
+      withLabel:'mc_large'{
+        cpus = { check_max( 8 * task.attempt, 'cpus' ) }
+        memory = { check_max( 16.GB * task.attempt, 'memory' ) }
+        time = 24.h
+      }
 
-  withLabel:'mc_huge'{
-     cpus = { check_max( 32 * task.attempt, 'cpus' ) }
-     memory = { check_max( 256.GB * task.attempt, 'memory' ) }
-     time = 24.h
-   }
+      withLabel:'mc_huge'{
+        cpus = { check_max( 32 * task.attempt, 'cpus' ) }
+        memory = { check_max( 256.GB * task.attempt, 'memory' ) }
+        time = 24.h
+      }
+    }
   }
+  raven {
+  // Specific nf-core/eager process configuration
+      params {
+        // Specific nf-core/configs params
+        config_profile_contact = 'James Fellows Yates (@jfy133)'
+        config_profile_description = 'nf-core/eager MPCDF raven profile provided by nf-core/configs'
+      }
+    process {
+
+      withName: malt {
+        maxRetries = 1
+        memory = 20000000.MB
```

Comment:
20TB memory, really?

Reply:
I wish...

Reply:
Also maybe if this is really true consder using at least the `.GB` unit, to make it easier to read ;-)

Reply:
Doing it for clarity as that' exactly wat we specify